lib/axios.js

```diff
@@ -63,7 +63,7 @@ const _error = (e) => {
   // Add original error thrown by the service, if exists
   const o = err.response?.data?.error ; if (!o) throw err
   err.message = !o.code || o.code == 'null' ? o.message : `${o.code} - ${o.message}`
-  // err.cause = o instanceof Error ? o : Object.assign (new Error, o)
```

Comment:
This line was disabled by @danjoa just a few hours ago arguing it _didn't help much_.
@johannes-vogel Can you provide an example where it does?

Reply:
I've adapted our usage here: https://github.tools.sap/cap/cds/commit/1543b03de2f3a6c69277ac9575c929075221ee68
It's not really needed but wanted to point out that this was potentially also used by others because it made the OData error response simpler accessible within the error object.